app/views/heat_asset_lists/_primary_table.html.haml

```diff
@@ -1,4 +1,6 @@
-%h2 Primary network
+%h2 Primary heat infrastructure
+.help-block
+  %p The table below lists the transport pipes between your sources and the feed-in point of the heat network in your LES. You can edit the cost aspects of this primary infrastructure here. The lengths of pipes are determined by the distance of the sources in the Heat source tab. More information can be found in the <a href='https://github.com/quintel/etmoses/wiki/Heat-network'>documentation</a>.
```

Comment:
If possible, aim for a maximum of 80 characters per line. You can split the content across multiple lines by intending another level:

```haml
.help-block
  %p
    The table below lists the transport pipes between your sources and the
    feed-in point of the heat network in your LES. You can edit the cost
    aspects of this primary infrastructure here. The lengths of pipes are
    determined by the distance of the sources in the Heat source tab. More
    information can be found in the documentation.

```